Plugin.php

```diff
@@ -97,4 +136,143 @@ public function boot()
             ]);
         });
     }
+
+    /**
+     * Extend the uploadable Widgets to support streaming file uploads directly to S3
+     */
+    protected function extendUploadableWidgets()
+    {
+        $addDependencies = function (WidgetBase $widget): void {
+            $widget->extendClassWith(SignedStorageUrlBehaviour::class);
+            $widget->addJs('/plugins/winter/driveraws/assets/js/build/stream-file-uploads.js');
+        };
+
+        MediaManager::extend($addDependencies);
+        FileUpload::extend($addDependencies);
+        RichEditor::extend($addDependencies);
+        MarkdownEditor::extend($addDependencies);
+    }
+
+    /**
+     * Hook into the backend.widgets.uploadable.onUpload event to process streamed file uploads
+     */
+    protected function processUploadableWidgetUploads()
+    {
+        Event::listen('backend.widgets.uploadable.onUpload', function (WidgetBase $widget): \Illuminate\Http\Response {
+            try {
+                /**
+                 * Expects the following input data:
+                 * - uuid: The unique identifier of uploaded file on S3
+                 * - name: The original name of the uploaded file
+                 * - path: The path to put the uploaded file (relative to the media folder and only takes effect if $widget->uploadPath is not set)
+                 */
+                $uploadedPath = 'tmp/' . Request::get('uuid');
+                $originalName = Request::get('name');
```

Comment:
@jaxwilko why are you accessing these input vars through `Request::get()` instead of `Request::input()` or even `Request::post()`?

Reply:
Think it's a copy paste from somewhere else that was using that style